state/protocol/badger/mutator_test.go

```diff
@@ -973,7 +973,8 @@ func TestExtendEpochSetupInvalid(t *testing.T) {
 			return setup, receipt, seal
 		}
 
```

Comment:
Should we also add a test to make sure epoch related metrics dont get updated while EECC is triggered ?

Reply:
Added in https://github.com/onflow/flow-go/pull/1569/commits/e31c7ecbf7b6adb8de573a5820008e2feed36029 https://github.com/onflow/flow-go/pull/1569/commits/152aa5ac69bb33302f6d36f58f2b071a68229f6f